code/modules/events/crystal_event.dm

```diff
@@ -256,6 +257,33 @@ This section is for the destabilized SM
 		active = FALSE
 		is_stabilized = TRUE
 
+/obj/machinery/destabilized_crystal/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>The Crystal appears to be heavily destabilized. Maybe it can be fixed by injecting it with something from another world.</span>"
+
+/obj/machinery/destabilized_crystal/Bumped(atom/movable/AM)
+	if(isliving(AM))
+		AM.visible_message("<span class='danger'>\The [AM] slams into \the [src] inducing a resonance... [AM.p_their()] body starts to glow and burst into flames before flashing into dust!</span>",\
+		"<span class='userdanger'>You slam into \the [src] as your ears are filled with unearthly ringing. Your last thought is \"Oh, fuck.\"</span>",\
+		"<span class='hear'>You hear an unearthly noise as a wave of heat washes over you.</span>")
+	else
+		return
+	playsound(get_turf(src), 'sound/effects/supermatter.ogg', 50, TRUE)
+	Consume(AM)
+
+/obj/machinery/destabilized_crystal/proc/Consume(atom/movable/AM)
+	if(isliving(AM))
+		var/mob/living/user = AM
+		if(user.status_flags & GODMODE)
+			return
+		message_admins("[src] has consumed [key_name_admin(user)] [ADMIN_JMP(src)].")
+		investigate_log("has consumed [key_name(user)].", INVESTIGATE_SUPERMATTER)
+		user.dust(force = TRUE)
+	for(var/obj/structure/crystal_portal/portals in GLOB.crystal_portals)
+		portals.spawn_time += 5 SECONDS
+	priority_announce("The sacrifice of a member of the station (we hope was the clown) has weakened the portals and the monsters generation is slowing down!")
+	sound_to_playing_players('sound/misc/notice2.ogg')
```

Comment:
This feels like it could be super spammy. What's stopping me from just throwing every pet/monkey/genetics human into this to spam announcements?

Also, doesn't `priority_announce` have its own sound parameter?

Reply:
Maybe I should add a check for sentience? Is there any helper like issentient()?

I always found the priority announcement like that since when I tryed to make the announcement without adding the sound nothing could be heard